Dockerfile.tools

```diff
@@ -7,7 +7,7 @@ WORKDIR /go/src/github.com/openshift/windows-machine-config-operator/
 
 COPY . .
 
-RUN yum -y update && yum -y install git make python2 python2-pip gcc
+RUN yum -y update && yum -y install git make python2 python2-pip gcc jq
```

Comment:
We should switch to python3 since python2 is EOLed

Reply:
That is outside the scope of this PR and should be done separately.